packages/devtools_server/docs/daemon.md

```diff
@@ -87,7 +89,9 @@ This request lists all DevTools instances that are currently connected back to t
 {
 	'id': '123',
 	// The `method` field is populated based on the ServiceRegistered VM event
-	'method': 's2.launchDevTools',
+	// if sending over the VM Service protocol.
+	// 'method': 's2.launchDevTools',
+	'method': 'devTools.launch',
```

Comment:
I think the json daemon variant - `devTools.launch` will also need a uri param (like the `vm.register` call).

Reply:
Good spot - I missed adding that. It's called `vmServiceUri` (`uri` seemed a bit ambiguous given the request is just "launch dev tools" and DevTools has its own URI). (in 04f1ac54)